apps/models/base/app_template.py

```diff
@@ -19,6 +19,7 @@ class Apps(models.Model):
     chart_archive = models.FileField(upload_to="apps/", null=True, blank=True)
     created_on = models.DateTimeField(auto_now_add=True)
     description = models.TextField(blank=True, null=True, default="")
+    gpu_enabled = models.BooleanField(default=False)
```

Comment:
```suggestion
    gpu_enabled = models.BooleanField(default=False, description="This flag is a Boolean field that, when set to True, indicates that the app template supports GPU. To actually allocate gpu you need to set the requested amount in k8s_values. Otherwise the default amount is zero")
```